lib/uds_plus_test_kit/uds_plus_test_suite.rb

```diff
@@ -20,7 +20,7 @@ class UDSPlusTestSuite < Inferno::TestSuite
 
         fhir_resource_validator do
             # The home-lab-report contains validation tools for certain codes missing in the UDS+ package
-            igs('fhir.hrsa.uds-plus#1.0.1', 'hl7.fhir.us.home-lab-report#1.0.0')
+            igs('fhir.hrsa.uds-plus#1.1.0', 'hl7.fhir.us.home-lab-report#1.0.0')
```

Comment:
This doesn't reference any files stored in the test kit anymore `igs/<packagename>.tgz`, which is great but means you should just delete all the files in the `/igs` directory to avoid any confusion.

Unless  `/igs/uds-plus-package-v110.tgz` contains altered content, in which case you'd want to update this igs line to point to that instead of the published version.

More information on declaring validators:
https://inferno-framework.github.io/docs/writing-tests/fhir-validation.html#defining-validators

Reply:
Understood! Packages have been removed.